Core/CoreTests/Inbox/ComposeMessage/ComposeMessageInteractorLiveTests.swift

```diff
@@ -153,4 +164,14 @@ class ComposeMessageInteractorLiveTests: CoreTestCase {
         stateUpdate.fulfill()
         wait(for: [stateUpdate], timeout: 1)
     }
+
+    class URLSessionDataTaskPublisherProviderMock: URLSessionDataTaskPublisherProvider {
```

Comment:
There are two versions of this mock and none of them are private.
Please either unite & extract it some place or make both of them private to potentially not cause issues down the line.